test-cypress/integration/home/language.change.spec.js

```diff
@@ -0,0 +1,44 @@
+import HomeSteps from '../../steps/home-steps';
+import SparqlSteps from '../../steps/sparql-steps';
+
+describe('Home screen language validation', () => {
+    let repositoryId;
+
+    beforeEach(() => {
+        cy.viewport(1280, 1000);
```

Comment:
Is there a reason for the view port resolution to be changed?

Reply:
This appeared in a few other tests. Wasn't sure if it had a significance that I wasn't getting.